src/ReactLeafletMap/index.js

```diff
@@ -27,13 +28,30 @@ class ReactLeafletMap extends React.PureComponent {
         super(props);
 
         this.state = {
-            view: null
+            view: null,
+            crs: this.getCRS(props.crs)
         }
 
         this.onLayerClick = this.onLayerClick.bind(this);
         this.onViewportChange = this.onViewportChange.bind(this);
     }
 
+    getCRS(code) {
+        switch (code) {
+            case 'EPSG:4326':
+                return L.CRS.EPSG4326;
+            case 'EPSG:5514':
+                return new Proj.CRS("EPSG:5514","+proj=krovak +lat_0=49.5 +lon_0=24.83333333333333 +alpha=30.28813972222222 +k=0.9999 +x_0=0 +y_0=0 +ellps=bessel +towgs84=589,76,480,0,0,0,0 +units=m +no_defs",
```

Comment:
Are we going to use the values for Krovak in other frameworks too? If so, they should be in constants.